blockdevice/cmd/wipe/main.go

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
```

Comment:
this command is not built by default, but it can be used to test the wiping on some real hardware